frame/alliance/src/lib.rs

```diff
@@ -407,7 +407,6 @@ pub mod pallet {
 		pub phantom: PhantomData<(T, I)>,
 	}
 
-	#[cfg(feature = "std")]
```

Comment:
Should be possible to replace the impl here with `derive(DefaultNoBound)`.

Reply:
done in: https://github.com/paritytech/substrate/pull/14108/commits/8a21eb158b0c03f2213fd8dfcafcaa9d0710397b